compiler/rustc_target/src/spec/x86_64_apple_darwin.rs

```diff
@@ -1,10 +1,12 @@
-use crate::spec::{LinkerFlavor, SanitizerSet, StackProbeType, Target, TargetOptions};
+use crate::spec::{
+    FramePointer, LinkerFlavor, SanitizerSet, StackProbeType, Target, TargetOptions,
+};
```

Comment:
Nit: could you avoid multi-line imports like this by splitting them into multiple `use`s?